README.md

```diff
@@ -40,3 +40,28 @@ There are two steps for uploading docs:
 
 For an example of using `docuploader`, see
 [example usage in googleapis/google-cloud-go](https://github.com/googleapis/google-cloud-go/blob/main/internal/kokoro/publish_docs.sh).
+
+## Requirements for DocFX YAML tarballs
```

Comment:
This looks good - is there anything else we should know about the path that docuploader uploads to?

Reply:
(Oh, and the file format - is it just tar, or tgz? I can't remember offhand. I'm sure I could check, but it would be good for this page to basically say: "docuploader performs these steps".